### PR TITLE
Basic RSP implementation

### PR DESCRIPTION
**Pull request by anacierdem**
Dockerizes the toolchain & adds some basic RSP functionality.

RSP syntax (only loads are implemented);

`loadVector vr, INSTRUCTION(element, offset), GPR`

vr: vector register
GPR: general purpose register

e.g.;

`loadVector $3, LBV(3, 0x10), $5`

### CODE REVIEW

File: tools/mksprite/Makefile [hunk 1]
```diff
@@ -1,10 +1,12 @@
 INSTALLDIR = $(N64_INST)
-CFLAGS = -std=gnu99 -O2 -Wall -Werror -I../../include -lpng
```
Comment (DragonMinded) [inline tools/mksprite/Makefile]: Why no -Werror? -Wall is pretty useless without it.

File: include/regsinternal.h [hunk 2]
```diff
@@ -146,7 +146,7 @@ typedef struct SP_regs_s {
     /** @brief RSP memory address (IMEM/DMEM) */
     volatile void * RSP_addr;
     /** @brief RDRAM memory address */
-    volatile void * RDAM_addr;
+    volatile void * DRAM_addr;
```
Comment (DragonMinded) [inline include/regsinternal.h]: LOL nice catch

**Comment by DragonMinded** [comment]
Can you comment on removing -Werror? This looks fine to merge in general by the way.

**Comment by anacierdem** [comment]
It was an artifact from @vieux's changes that I'm not aware of. Added it back :)

**Comment by anacierdem** [comment]
Also you may want to edit the readme afterwards FYI.

**Pull request merged**